pkgs/top-level/all-packages.nix

```diff
@@ -31399,6 +31399,8 @@ with pkgs;
 
   manuskript = libsForQt5.callPackage ../applications/editors/manuskript { };
 
+  minari = python3Packages.toPythonApplication python3Packages.minari;
```

Comment:
Is there a `meta.mainProgram` automatically set for this?

Reply:
Not sure but it works.
`manim` is in the same case and does not declare a `mainProgram`.

Reply:
I have found other python packages doing it so I added it.

Reply:
> Does running `nix run .#minari` works? If yes, then it's fine.

This was working without and still is.

Reply:
Right, let's leave it like this, it's fine now.